src/components/RecentBlog/index.module.css

```diff
@@ -0,0 +1,34 @@
+.blogItem {
+    margin-bottom: 1.5rem;
+    padding: 1rem;
+    border: 1px solid #ddd;
+    border-radius: 8px;
+    transition: box-shadow 0.3s;
+    background-color: #fff;
+}
+
+.blogItem:hover {
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+}
+
+.blogTitle {
+    font-size: 1.25rem;
+    font-weight: bold;
+    color: #333;
+    text-decoration: none;
+}
+
+.blogTitle:hover {
+    text-decoration: underline;
+}
+
+.blogMeta {
+    display: block;
+    margin-top: 0.5rem;
+    color: #666;
+}
+
+.blogDescription {
+    margin-top: 0.75rem;
+    color: #444;
+}
```

Comment:
This styling doesn't look good in the dark mode. We really need to use [Figma variables](https://github.com/facebookincubator/infima/blob/main/packages/core/styles/common/variables.pcss) for most of the properties like colors, shadows, font sizes, etc.